modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v5_3_2/TemplateUpgradeProcess.java

```diff
@@ -45,88 +49,123 @@ private String _getVariableName(Matcher matcher) {
 		return variableName.trim();
 	}
 
-	private String _replaceTemplatePattern(
```

Comment:
@velosofelipe64, this method shouldn't be removed, since it was added as a request from the Product Team review. Can you please put it back?